docker/indexer/storage/storage.go

```diff
@@ -18,22 +18,86 @@ package storage
 
 import (
 	"context"
+	"fmt"
+	"sync"
+	"time"
 
+	"cloud.google.com/go/datastore"
 	"github.com/go-git/go-git/v5/plumbing"
 	"github.com/google/osv.dev/docker/indexer/stages/preparation"
 	"github.com/google/osv.dev/docker/indexer/stages/processing"
+)
 
-	log "github.com/golang/glog"
+const (
+	kind   = "source-repository"
```

Comment:
Can you name this RepoIndex or something similar. We name all of our entities in that casing as well.

Reply:
This is also a bit too similar to our existing SourceRepository entity.